Sources/MeiliSearch/Model/KeyParams.swift

```diff
@@ -4,7 +4,7 @@ import Foundation
  `KeyParams` contains all the parameters to create an API key.
  */
 public struct KeyParams: Codable, Equatable {
-  public let description: String
+  public var description: String?
```

Comment:
Just out of curiosity, why are some var and some let? We cant change `actions` ? Is this because of `updateKey` ?

Reply:
Correct!